### PR TITLE
Add github action

### PR DESCRIPTION
**Pull request by pvalsecc**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        run: make build
+
+      - name: Acceptance
+        run: make acceptance
+
+      - name: Send coverage
+        if: always()
+        run: make send_coverage
+        env:
+          CODACY_PROJECT_TOKEN: ${{secrets.CODACY_PROJECT_TOKEN}}
```
Comment (sbrunner) [inline .github/workflows/main.yml]: Do we steal want to use Codacy?

File: .github/workflows/rebuild.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+      - name: mypy
+        run: make mypy
+
+      - name: install GDAL
+        run: docker run --rm camptocamp/c2cwsgiutils:latest-full /opt/c2cwsgiutils/install_gdal.sh
+
+      - name: Release
+        run: |
+          # TODO: docker login
```
Comment (sbrunner) [inline .github/workflows/rebuild.yml]: Why not to do it directly

File: .github/workflows/main.yml (hunk 1 above)
Comment (pvalsecc) [reply .github/workflows/main.yml]: Why not?

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    steps:
+      - uses: actions/checkout@v1
+
+      - uses: camptocamp/initialise-gopass-summon-action@v1
+        with:
+          large-secret-passphrase: ${{secrets.LARGE_SECRET_PASSPHRASE}}
+          github-gopass-ci-token: ${{secrets.GITHUB_GOPASS_CI_TOKEN}}
+
+      - name: Pull
+        run: make pull
```
[the remark this answers is not in the document]
Comment (pvalsecc) [reply .github/workflows/main.yml]: We'll need to pull the images anyway. That way, it's split in another setp...

File: .github/workflows/rebuild.yml (hunk 2 above)
Comment (pvalsecc) [reply .github/workflows/rebuild.yml]: Oops, I forgot to adapt this file too. Let me fix that

**Pull request merged**